hamster_gtk/overview/dialogs.py

```diff
@@ -109,6 +109,9 @@ def refresh(self):
         self._grouped_facts, self._totals = self._group_facts()
 
         helpers.clear_children(self.main_box)
+        if self._charts:
```

Comment:
You are right, should be unnecessary to destroy it again.
Even if you destroy the widget, the associated variables will still contain the widget – destroy only cleans the Gtk objects, the corresponding Python objects remain for GC to clear them so we have to set it to False.